packages/developer-extension/src/background/domain/logRequests.ts

```diff
@@ -0,0 +1,64 @@
+import { store } from '../store'
+
+const decoder = new TextDecoder('utf-8')
+chrome.webRequest.onBeforeRequest.addListener(
+  (info) => {
+    if (!store.logEvents) {
+      return
+    }
+    if (info.tabId < 0) {
+      console.log('Some intake request was made in a non-tab context... (service worker maybe?)')
+      return
+    }
+
+    const url = new URL(info.url)
+
+    const intake = url.hostname.match(/^\w*/)?.[0]
+    if (!intake) {
+      return
+    }
+    if (!info.requestBody.raw) {
+      return
+    }
+
+    for (const rawBody of info.requestBody.raw!) {
+      if (rawBody.bytes) {
+        const decodedBody = decoder.decode(rawBody.bytes)
+        for (const rawEvent of decodedBody.split('\n')) {
+          const event = sortProperties(JSON.parse(rawEvent))
+          chrome.tabs.executeScript(info.tabId, {
+            code: `console.info("Browser-SDK:", ${JSON.stringify(intake)}, ${JSON.stringify(event)});`,
+          })
+        }
+      }
+    }
+  },
+  {
+    urls: [
+      // TODO: implement a configuration page to add more URLs in this list.
+      'https://*.logs.datadoghq.com/*',
+      'https://*.browser-intake-datadoghq.com/*',
+      'https://*.logs.datadoghq.eu/*',
+      'https://*.browser-intake-datadoghq.eu/*',
```

Comment:
*.logs.datadoghq.com is also intercepting non sdk requests (ex: livetail), what about being a bit more restrictive?